components/azure/azureml/aml-deploy-model/src/deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Deploy registered model to Azure Machine Learning
+while getopts "n:m:i:d:s:p:u:r:w:t:" option;
+    do
+    case "$option" in
+        n ) DEPLOYMENT_NAME=${OPTARG};;
+        m ) MODEL_NAME=${OPTARG};;
+        i ) INFERENCE_CONFIG=${OPTARG};;
+        d ) DEPLOYMENTCONFIG=${OPTARG};;
+        s ) SERVICE_PRINCIPAL_ID=${OPTARG};;
+        p ) SERVICE_PRINCIPAL_PASSWORD=${OPTARG};;
+        u ) SUBSCRIPTION_ID=${OPTARG};;
+        r ) RESOURCE_GROUP=${OPTARG};;
+        w ) WORKSPACE=${OPTARG};;
+        t ) TENANT_ID=${OPTARG};;
+    esac
+done
+az login --service-principal --username ${SERVICE_PRINCIPAL_ID} --password ${SERVICE_PRINCIPAL_PASSWORD} -t ${TENANT_ID}
```

Comment:
Set subscription post login in case service principal having access to multiple subscriptions  
az account set --subscription <subscription_id>

Reply:
changed.